Jenkinsfile

```diff
@@ -76,3 +95,15 @@ void withMavenEnv(List envVars = [], def body) {
         body.call()
     }
 }
+
+void renameFiles(def files, String prefix) {
+    for(i = 0; i < files.length; i++) {
```

Comment:
I'm pretty sure it'd be executed on the agent, but yeah, `File` isn't whitelisted, so this is probably for the best.